salt/post_provisioning_cleanup.sh

```diff
@@ -31,13 +42,25 @@ elif [ -x /usr/bin/apt ]; then
     INSTALLER=apt
 fi
 
-echo "Removing Salt packages, except Salt Bundle (venv-salt-minion) ..."
+echo "Removing ${PKGS} packages, except Salt Bundle (venv-salt-minion) ..."
 if [[ "$INSTALLER" == "zypper" ]]; then
-zypper -q --non-interactive remove salt-minion > /dev/null 2>&1 ||:
+    if [[ "$SERVER" == "true" ]];then
+        zypper -q --non-interactive remove salt-minion > /dev/null 2>&1 ||:
+    else
+        zypper -q --non-interactive remove salt salt-minion python3-salt python2-salt > /dev/null 2>&1 ||:
+    fi
 elif [[ "$INSTALLER" == "yum" ]]; then
-yum -y remove salt salt-minion python3-salt python2-salt > /dev/null 2>&1 ||:
+    if [[ "$SERVER" == "true" ]];then
+        yum -y remove salt-minion > /dev/null 2>&1 ||:
+    else
+        yum -y remove salt salt-minion python3-salt python2-salt > /dev/null 2>&1 ||:
+    fi
 elif [[ "$INSTALLER" == "apt" ]]; then
-apt-get --yes purge salt-common > /dev/null 2>&1 ||:
+    if [[ "$SERVER" == "true" ]];then
+        apt-get --yes purge salt-client > /dev/null 2>&1 ||:
```

Comment:
I think it should be `salt-minion` here, right?

```suggestion
        apt-get --yes purge salt-minion > /dev/null 2>&1 ||:
```